icu4c/APIChangeReport.md

```diff
@@ -1,642 +1,92 @@
 
-  
 <!--
- Copyright © 2019 and later: Unicode, Inc. and others.
+ Copyright © 2019: Unicode, Inc. and others.
  License & terms of use: http://www.unicode.org/copyright.html
 -->
 
-# ICU4C API Comparison: ICU 64 (update #1: 64.2) with ICU 65
+# ICU4C API Comparison: ICU 65 with ICU 66 (m1)
```

Comment:
preview, not m1

Reply:
This is auto generated based on the version number, but I can manually change it?